lib/Plugin.class.php

```diff
@@ -53,6 +53,7 @@ public function instantiateCompiler()
         $this->compiler = new WPLessCompiler;
         $this->compiler->setVariable('stylesheet_directory_uri', "'" . get_stylesheet_directory_uri() . "'");
         $this->compiler->setVariable('template_directory_uri', "'" . get_template_directory_uri() . "'");
+        if(!WP_DEBUG) { $this->compiler->setFormatter('compressed'); }
```

Comment:
Also, I think it would make sense to set the formatter to *compressed* if:
- if `WP_DEBUG` is not defined (do we need to use `defined`?)
- if no formatter has been set yet (we don't want to override one which has already been set by the user)

Reply:
```suggestion
        if(!defined('WP_DEBUG') || !WP_DEBUG) {
      		$this->compiler->setFormatter('compressed');
        }
```